plugins/bundle/plugin.go

```diff
@@ -157,7 +157,10 @@ func (p *Plugin) Reconfigure(ctx context.Context, config interface{}) {
 	// Look for any bundles that have had their config changed, are new, or have been removed
 	newConfig := config.(*Config)
 	newBundles, updatedBundles, deletedBundles := p.configDelta(newConfig)
+
+	p.mtx.Lock()
```

Comment:
There already is a `p.cfgMtx` that is held for this purpose further above

Reply:
Yes, thanks for pointing out.